main.nf

```diff
@@ -1,6 +1,6 @@
 #! /usr/bin/env nextflow
 // Enabling DSL2
-nextflow.enable.dsl=2
+// nextflow.enable.dsl=2
```

Comment:
Good, point!

The `stub` feature isn't dependent on `dsl2`.

As a best practice in any PR, you should include a minimal and sufficient number of changes, therefore for this `stubs` PR it's not relevant and this line can be removed.

Reply:
Ok! i'll focus on just the stubs creation.

Reply:
Cool, please remove this line from the current PR.